webapp/src/routes/account/index.js

```diff
@@ -21,7 +21,7 @@ import CloseIcon from '@material-ui/icons/Close'
 import _get from 'lodash.get'
 
 import TitlePage from 'components/title-page'
-import config from 'config'
```

Comment:
I change this import because it becames confuse to know where those variables are taken